tedge/src/bin/tedge_priv.rs

```diff
@@ -0,0 +1,27 @@
+fn main() -> anyhow::Result<()> {
+    eprintln!("Privileged tedge process");
+    // assert!(users::get_current_uid() == 0);
+
+    loop {
+        let mut line = String::new();
+        let _ = std::io::stdin().read_line(&mut line)?;
+
+        eprintln!("Received command: {}", line);
+
+        match line.trim() {
+            "command1" => {
```

Comment:
In practice, you might need something much more complex here. For example, if you want to include some parameter to the command which includes newlines then using newline to separate commands might not be sufficient. I think it's worth understanding what protocol we would use between these processes to allow for future flexibility.